resource/resource.go

```diff
@@ -121,22 +121,7 @@ func (r *Resource) Check(
 	env oc.Environment,
 	log *oc.Logger,
 ) ([]oc.Version, error) {
```

Comment:
we cannot remove this method now because it is needed to implement the ofcourse.Resource interface.